lib/tasks/batch_connect.rake

```diff
@@ -0,0 +1,31 @@
+namespace :batch_connect do
+  desc "Generate new batch connect session"
+  task new_session: :environment do
+    # Read in user settings
+    app = ENV["BC_APP_TOKEN"] || abort("Missing environment variable BC_APP_TOKEN")
+    fmt = ENV["BC_RENDER_FORMAT"]
+    ctx = if !$stdin.tty?
```

Comment:
In tools that accept a file or stdin, if the filename provided is `-` then it reads from stdin. Unfortunately rake isn't the easiest to supply arguments. So another solution would be to read from stdin if `BC_SESSION_CONTEXT=-`.